### PR TITLE
TG-154: profile screen and edit personal info

### PR DESCRIPTION
**Pull request by ilcosme**
Updated form to show only the necessary fields according to changes in employee entity, in both sign up and edit. All functionalities working.

In this branch we'll work also the necessary routing to complete the flow of the app.

### CODE REVIEW

File: src/Components/AdminFlow/Profile/Profile.jsx [hunk 1]
```diff
@@ -43,8 +32,10 @@ function Profile() {
     mode: 'onBlur',
     resolver: joiResolver(schema)
   });
+  console.log(errors);
```
Comment (FefeTroanes) [inline src/Components/AdminFlow/Profile/Profile.jsx:35]: fijate que te quedo un cosologueo

File: src/Components/AdminFlow/Profile/Profile.jsx [hunk 2]
```diff
@@ -43,8 +32,10 @@ function Profile() {
...
   const UpdateAdmin = (data) => {
-    dispatch(thunksAdmins.updateAdmin(data, param.id));
+    console.log(data);
```
Comment (FefeTroanes) [inline src/Components/AdminFlow/Profile/Profile.jsx:37]: otro

File: src/Components/Login/index.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ import Loader from 'Components/Shared/Loader';
 
 const Login = () => {
   const history = useHistory();
-  const role = useSelector((state) => state.auth.authenticated?.role);
+  //const role = useSelector((state) => state.auth.authenticated?.role);
```
Comment (FefeTroanes) [inline src/Components/Login/index.js:14]: esto que quedo comentado creo que podria borrarse

File: src/Components/Shared/Sidebar/sidebar.module.css [hunk 4]
```diff
@@ -1,116 +1,130 @@
...
 }
 
 @media screen and (min-width: 1024px) {
-    .Sidebar {
-        width: 20vw;
-    }
+  .Sidebar {
+    width: 20vw;
+  }
 }
```
Comment (FefeTroanes) [inline src/Components/Shared/Sidebar/sidebar.module.css:130]: aca ya que estamos github pide una linea al final

**Review by FefeTroanes — COMMENTED** [review]
😎👉🏼

**Review by Clarosluciano — APPROVED** [review]

**Review by FefeTroanes — APPROVED** [review]
ads

**Pull request merged**